samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -0,0 +1,263 @@
+/*
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* More information about these options at jshint.com/docs/options */
+
+/* globals trace, InfoBox */
+/* exported AppController */
+
+'use strict';
+
+function $(id) {
+  return document.querySelector('#' + id);
+}
```

Comment:
@samdutton added this same method with a slightly different semantic. 
+function $(selector){
+  return document.querySelector(selector);
+}